ops/scripts/tag-bedrock-go-modules.sh

```diff
@@ -36,6 +36,9 @@ go get github.com/ethereum-optimism/optimism/op-service@$VERSION
 go get github.com/ethereum-optimism/optimism/op-chain-ops@$VERSION
 go mod tidy
 
```

Comment:
Should we make a script that autogens `version.go`?

Reply:
Probably, but we use a different type of versionning between op-node and op-proposer/op-batcher